adds website redesign and survey reminder (#545)
ember-learn/ember-blog: source/2020-02-28-the-ember-times-issue-137.md

---

## [Ember Website Redesign 🔥](https://emberjs.com/)

> Maybe you haven't noticed, but the **primary website for Ember** has been **redesigned** and it is **now live**! The redesign brings a fresh new look at the main piece of real estate for Ember on the interent and features a clean and modern look that all of us in the community can be proud of. 

Reviewer: ```suggestion
Maybe you haven't noticed, but the **primary website for Ember** has been **redesigned** and it is **now live**! The redesign brings a fresh new look at the main piece of real estate for Ember on the internet and features a clean and modern look that all of us in the community can be proud of. 
```

 small typo :)

> This redesign has been a long time in the making and has invovled a lot of work from the people across the Ember teams and the Ember community. Big shout outs to everyone involved in the effort, especially [Mel Sumner @melsumner](https://github.com/MelSumner) and [Chris Manson @mansona](https://github.com/mansona)! 🎉🎉🎉 It's super exciting to see this modern re-visioning of the website come to life.

Reviewer: ```suggestion
This redesign has been a long time in the making and has invovled a lot of work from the people across the Ember teams and the Ember community. Big shout outs to everyone involved in the effort, especially [Mel Sumner @melsumner](https://github.com/MelSumner) and [Chris Manson @mansona](https://github.com/mansona)! 🎉🎉🎉 It's super exciting to see this modern vision of the website come to life.
```
For me re-visioning feels like a really complex word which I cannot completely place, so vision might be easier

> If you are enjoying the new website and would like to help make all the Ember web properties have a consistent design, please let the Learning Team know on Discord in the #dev-ember-learning channel. 

Reviewer: Maybe we could add the link to the discord server even though I would prefer a direct link to the channel, but I couldn't find it

Author: > I've some small comments and ideas :)

Thank you @MinThaMie ! I committed all of your suggestions. 😄

Author: yeah i updated to make it consistent with the linking in the community survey reminder copy below

> If you notice anything wrong with the website please also feel free to open an issue in either the [main website repo](https://github.com/ember-learn/ember-website/) or the [styleguide](https://github.com/ember-learn/ember-styleguide) where the components live. 

Reviewer: ```suggestion
If you notice anything wrong with the website please also feel free to open an issue in either the [main website repo](https://github.com/ember-learn/ember-website/) or the [styleguide repo](https://github.com/ember-learn/ember-styleguide) where the components live. 
```

---

## [2020 Ember Community Survey 📝](https://tilde.wufoo.com/forms/2020-emberjs-community-survey/)

Here's another friendly reminder that it’s that time of year again 😀, the **6th annual official 2020 Ember Community Survey** is here! Once again we would love your help to learn about who is in the Ember community and how they work with the framework

Last year over 1200 people participated in the survey. All that participation and your participation over the years has resulted in enough input to make the process more efficient. This year the survey has been considerably slimmed down from previous years! So it should be easier than ever to make a **contribution** to our community by filling out the survey – let’s keep that participation going! 🎉

This year we’d also like to emphasize that contributions from non-Ember users are valuable as well. So if you have co-workers or friends who you think might be interested in filling out the survey, please share it with them.

Submissions will be accepted **until March 8th**, but there’s no need to wait! Go ahead and fill out the [survey here now](https://tilde.wufoo.com/forms/2020-emberjs-community-survey/). 🔥🔥🔥

> We look forward to your participation! If you have any immediate questions, feel free to email the survey team via survey@emberjs.com, or ping us in #dev-ember-learning on the [Discord chat](https://discordapp.com/invite/emberjs).

Reviewer: ```suggestion
We look forward to your participation! If you have any immediate questions, feel free to email the survey team via survey@emberjs.com, or ping them in #dev-ember-learning on the [Discord chat](https://discordapp.com/invite/emberjs).
```

I assume that the us is a reference to the survey team, and it feels like that is not us in this case?
